src/mame/heathkit/h89.cpp

```diff
@@ -329,6 +334,13 @@ static INPUT_PORTS_START( h89 )
     PORT_DIPSETTING( 0x00, DEF_STR( Normal ) )
     PORT_DIPSETTING( 0x80, "Auto" )
 */
+
+	PORT_START("CONFIG")
+	PORT_CONFNAME(0x03, 0x00, "CPU Clock Speedup Upgrade")
+	PORT_CONFSETTING(0x00, DEF_STR( None ) )
+	PORT_CONFSETTING(0x01, "2 / 4 MHz")
+	PORT_CONFSETTING(0x02, "2 / 6 MHz")
+	PORT_CONFSETTING(0x03, DEF_STR( None ) )
```

Comment:
Better not to have two redundant settings for no speed-up.  It’s fine not to expose every bit pattern for these configuration inputs.